src/lib/email.ts

```
@@ -0,0 +1,124 @@
+
+  try {
+    const data = await resend.emails.send({
+      from,
+      to,
+      subject,
+      html,
+    });
+
+    return data;
+  } catch (error) {
+    throw error;
```

Biome (1.9.4)

[error] The catch clause that only rethrows the original error is useless.

An unnecessary catch clause can be confusing.
Unsafe fix: Remove the try/catch clause.


(lint/complexity/noUselessCatch)